src/Filter/TypeCastInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas-api-tools/api-tools-doctrine-querybuilder for the canonical source repository
+ * @copyright https://github.com/laminas-api-tools/api-tools-doctrine-querybuilder/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas-api-tools/api-tools-doctrine-querybuilder/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\ApiTools\Doctrine\QueryBuilder\Filter;
+
+interface TypeCastInterface
+{
+    public function typeCastField($metadata, $field, $value, $format = null, $doNotTypecastDatetime = false);
```

Comment:
Please provide docblock annotations for each parameter. We can also bump to a PHP 7 version with this release, allowing you to add explicit typehints where applicable (e.g., `string $field`, `?string $format = null`, `bool $doNotTypecastDatetime = false`).

Reply:
:+1: